scripts/gen.py

```diff
@@ -819,9 +824,19 @@ def generate_aten_to_xla(ctx, tree, rwxtree, fname, sig, rwsig, params, fnopts):
       xla_ref_param = param_vars[-1]
   code += tfetcher.generate_fetches()
   result_assign = generate_result_assignment(tree, _RESULT_NAME)
-  code += '  {}{};\n'.format(
-      result_assign, get_handling_function(ctx, fname, xla_ref_param,
-                                           param_vars))
+  # TODO(https://github.com/pytorch/pytorch/issues/40250):
```

Comment:
I was more like https://github.com/pytorch/xla/issues/2240
It is hard to track issues on another repo 😉